### PR TITLE
move btiwise module to silver

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by ryanchou-dev** [comment]
resolves #3704

**Review by envyaims — APPROVED** [review]
ORZZZ

**Comment by SansPapyrus683** [comment]
image

💀

**Comment by ryanchou-dev** [comment]
fixing in a sec sorry

File: content/3_Silver/Intro_Bitwise.mdx
Comment (bqi343) [inline content/3_Silver/Intro_Bitwise.mdx:1]: add a redirect to the frontmatter. example:

Screen Shot 2023-06-05 at 9 44 42 PM

**Comment by SansPapyrus683** [comment]
image

mfw

**Comment by SansPapyrus683** [comment]
image
what in the

File: content/3_Silver/Intro_Bitwise.mdx
Comment (bqi343) [reply content/3_Silver/Intro_Bitwise.mdx:1]: oh apparently there are already redirects. add the redirect below the previous ones

**Pull request merged**